Userland/Libraries/LibWeb/Geometry/DOMPointReadOnly.cpp

```diff
@@ -24,6 +25,11 @@ DOMPointReadOnly::DOMPointReadOnly(JS::Realm& realm, double x, double y, double
     set_prototype(&Bindings::cached_web_prototype(realm, "DOMPointReadOnly"));
 }
 
+JS::NonnullGCPtr<DOMPointReadOnly> DOMPointReadOnly::from_point(JS::VM& vm, Web::Geometry::DOMPointInit dom_point_init)
+{
+    return construct_impl(*vm.current_realm(), dom_point_init.x, dom_point_init.y, dom_point_init.z, dom_point_init.w);
```

Comment:
`// The fromPoint(other) static method on DOMPointReadOnly must create a DOMPointReadOnly from the dictionary other.`